en/option/series/scatter.md

```diff
@@ -90,11 +90,11 @@ Configurations of emphasis state.
     prefix = "##"
 ) }}
 
-### scale(boolean) = true
+### scale(boolean | number) = true
```

Comment:
`space` in the type section should be avoided. or it will not be parsed correctly.

```md
### scale(boolean|number) = true
```

Reply:
fixed